snippets/facets.liquid

```diff
@@ -261,7 +261,7 @@
               {% endcase %}
             {%- endfor -%}
             <noscript>
-              <button type="submit" class="facets__button-no-js button button--tertiary">{{ 'products.facets.filter_button' | t }}</button>
+              <button type="submit" class="facets__button-no-js button button--secondary">{{ 'products.facets.filter_button' | t }}</button>
```

Comment:
We will need to apply this to a few more buttons. I included a table in a main comment that indicates where we may need to change primary and tertiary to secondary.